akd_core/src/ecvrf/ecvrf_impl.rs

```diff
@@ -90,6 +91,15 @@ pub struct VRFExpandedPrivateKey {
     pub(super) nonce: [u8; 32],
 }
 
+impl Drop for VRFExpandedPrivateKey {
```

Comment:
Actually, after some thought, let's keep it the way it is now, since I feel like I "trust" that the drop is actually happening more when it is explicitly implemented, rather than relying on the zeroize_derive feature of the zeroize crate. But in the future if we end up putting this in more places, we can revisit using the derive macro.